document/core/exec/numerics.rst

```diff
@@ -626,6 +804,214 @@ The integer result of predicates -- i.e., :ref:`tests <syntax-testop>` and :ref:
    \end{array}
 
 
+.. _op-ibitselect:
+
+:math:`\ibitselect_N(i_1, i_2, c)`
+..................................
+
+* Use the bits in :math:`c` to select corresponding bit from :math:`i_1` when 1 and :math:`i_2` when 0.
```

Comment:
Maybe rephrase this? I can understand not wanting to write out the exact behavior, but I think it should still match the "Return ..." structure.

Reply:
Yea agree that this should be rephrased, tried something, but it still looks awkward, especially since the implementation below doesn't match the description.